test/test.js

```diff
@@ -387,10 +385,14 @@ describe( 'Utility Methods: ', function() {
 		process.argv[2] = '-c'
 		process.argv[3] = '.stylintrc'
 		var testMethod = app.setConfig( '.stylintrc' )
-		var testConfig = JSON.parse( stripJsonComments( fs.readFileSync( process.cwd() + '/.stylintrc', 'utf-8' ) ) )
+		var testConfig = app.setConfig( process.cwd() + '/.stylintrc' )
+
+		it( 'return complete config when not complete config is passed in', function() {
```

Comment:
Nice test! Can we paraphrase a bit to avoid ambiguity with what `complete` means here?

Perhaps something like:
```js
it( 'return config with all properties when passed config with missing properties', function() {
```

What do you think?

Reply:
Sounds good, I'll update the PR. Thnx.